src/credentials/attestation_verifier/DefaultDeviceAttestationVerifier.cpp

```diff
@@ -458,6 +458,50 @@ void DefaultDACVerifier::VerifyAttestationInformation(const DeviceAttestationVer
         VerifyOrExit(attestationError == AttestationVerificationResult::kSuccess, attestationError = attestationError);
     }
 
+    {
+        uint8_t issuerBuf[kMaxCertificateDistinguishedNameLength] = { 0 };
+        MutableByteSpan paaIssuer(issuerBuf);
+        MutableByteSpan paiIssuer(issuerBuf);
+        MutableByteSpan dacIssuer(issuerBuf);
```

Comment:
This reuse of the same buffer for ostensibly different things is very odd.

More generally, it seems like the same code is copy/pasted several times below, with some changes to the exact error codes.  Feels like there should be some helper function that takes the der buffer and returns success or error that we call three times...

Reply:
Agreed, will address this.